docker/docker_update.sh

```diff
@@ -1,6 +1,53 @@
 #!/bin/sh
 set -e
 
+Help()
+{
+  # Display help
+  echo "Usage: ./docker_update.sh [OPTIONS]"
+  echo ""
+  echo "Start all docker containers required to run Lemmy."
+  echo ""
+  echo "Options:"
+  echo "-u Docker username. Only required if managing Docker via Docker Desktop with a personal access token."
+  echo "-h Print this help."
+}
+
+while getopts ":hu:" option; do
+  case $option in
+    h) Help
+       exit;;
+    u) DOCKER_USER=$OPTARG
+       ;;
+    *) echo "Invalid option $OPTARG."
+       exit;;
+  esac
+done
+
+LOG_PREFIX="[🐀 lemmy]"
+ARCH=$(uname -m 2>/dev/null || echo 'unknown') # uname may not exist on windows machines; default to unknown to be safe.
+
 mkdir -p volumes/pictrs
+
+echo "$LOG_PREFIX Please provide your password to change ownership of the pictrs volume."
 sudo chown -R 991:991 volumes/pictrs
-sudo docker compose up -d --build
+
+if [ "$ARCH" = 'arm64' ]; then
+  echo "$LOG_PREFIX WARN: If building from images, make sure to uncomment 'platform' in the docker-compose.yml file!"
```

Comment:
This echo can be removed once https://github.com/LemmyNet/lemmy/pull/2806 is complete.